File: Droid3/app/src/main/java/com/example/acty/droid3/DBObjects/Campaign.java
package com.example.acty.droid3.DBObjects;

public class Campaign extends CampaignBase {
    private CampaignMedia campaignVisualResource;

    public CampaignMedia getCampaignVisualResource() {
        return campaignVisualResource;
    }

    public void setCampaignVisualResource(CampaignMedia campaignVisualResource) {
        this.campaignVisualResource = campaignVisualResource;
    }
}

File: Acty-master/Acty.Shared/DataObjects/Campaign.cs
using Newtonsoft.Json;

namespace Acty
{
    public enum CampaignCategory
    {
        Environmental,
        Social,
        Political,
        Economical,
        Local
    }

    public enum CampaignStatus
    {
        Completed,
        InProgress,
        OnHold,
        Active,
        Suspended,
        Cancelled,
        Flagged
    }

    public class Campaign : CampaignBase
    {
        [JsonProperty(PropertyName = "CampaignVisualResource")]
        public CampaignMedia CampaignVisualResource { get; set; }
    }

    public class DBCampaign : CampaignBase
    {
        [JsonProperty(PropertyName = "StoryMediaResourceBlob")]
        public string StoryMediaResourceBlob
        {
            get;
            set;
        }

        [JsonProperty(PropertyName = "id")]
        public string id { get; set; }
    }
}